RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -179,24 +179,24 @@ class RNUnifiedContacts: NSObject {
     mutableContact.familyName = contactData["familyName"] as! String
     mutableContact.organizationName = contactData["organizationName"] as! String
     
-    for number in contactData["phoneNumbers"] as! NSArray {
-      let transformedNumber = transformNumber( number as! NSDictionary )
-      mutableContact.phoneNumbers.append( transformedNumber )
+    for phoneNumber in contactData["phoneNumbers"] as! NSArray {
+      let phoneNumberAsNSDictionary = convertPhoneNumberToNSDictionary( phoneNumber as! NSDictionary )
+      mutableContact.phoneNumbers.append( phoneNumberAsNSDictionary )
     }
     
-    for email in contactData["emailAddresses"] as! NSArray {
-      let transformedEmail = transformEmail ( email as! NSDictionary )
-      mutableContact.emailAddresses.append( transformedEmail )
+    for emailAddress in contactData["emailAddresses"] as! NSArray {
+      let emailAddressAsNSDictionary = convertEmailAddressToNSDictionary ( emailAddress as! NSDictionary )
```

Comment:
Same thing here: 

`emailAddressAsCNLabeledValue` and `convertEmailAddressToCNLabeledValue`.